docs/releases/1.9.txt

```diff
@@ -919,6 +919,12 @@ versions:
 Its parsing caused bugs with the current syntax, so support for the old syntax
 will be removed in Django 2.0 following an accelerated deprecation.
 
+``ForeignKeyField`` and ``OneToOneField`` changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``on_delete`` argument of ``ForeignKeyField`` and ``OneToOneField`` will be
+required in Django 2.0.
```

Comment:
Some discussion of adding the option to historical migrations to avoid warnings would be useful.